examples/llm/mcloud/mcli-hf-generate.yaml

```diff
@@ -0,0 +1,24 @@
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_branch: release/v0.0.4 # use your branch
+  # git_commit: # OR use your commit hash
+  pip_install: -e .[llm]
+  ssh_clone: false # Should be true if using a private repo
+
+command: |
+  cd examples/examples/llm/inference
+  pip install awscli
+  aws s3 cp --recursive s3://bucket/folder/hf/ local_hf_folder
+  python hf_generate.py \
+    --name_or_path local_hf_folder \
+    --max_new_tokens 512
+
+image: mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04
+optimization_level: 0
+
+# Mosaic Cloud will use run_name (with a unique suffix) to populate the env var $COMPOSER_RUN_NAME
+run_name: hf-generate
+
+gpu_num: 8
```

Comment:
this is only single gpu so maybe only request 1 gpu here? i know that doesn't work on some of our clusters, so feel free to ignore this one and leave as is

Reply:
Will leave as 8 for now because I think most of the time we can't use just 1 GPU